### DISCUSSION TITLE
Multi model locations

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [lllyasviel/stable-diffusion-webui-forge#714] [Ideas]
Would it be possible to make forge read from multiple locations? As i now use 2 drives models. SD on one, SDXL on another. but it seems to only allow one model folder.

**Comment by svenjuga** [comment]
Here is, what did. Maybe it will help you too.
Inside the webui-user.bat you can define the checkpoint folder, its starts like "--ckpt ..." (remove @REM at the beginning of the line). 
You're meant to use this argument if you have Auto1111 installed already, so forge can load models from there. But, additionally, you can put models in the regular forge checkpoint folder "forge\models\stable diffusion\". Forge will list the checkpoints of both folders.
